manifests/install/repos/extra.pp

```diff
@@ -30,12 +30,7 @@
   }
 
   if $ipa_authentication and $configure_ipa_repo {
-    yumrepo { 'adelton-identity':
-      enabled  => 1,
-      gpgcheck => 0,
-      baseurl  => "http://copr-be.cloud.fedoraproject.org/results/adelton/identity_demo/epel-${osreleasemajor}-\$basearch/",
-      before   => Package['mod_authnz_pam', 'mod_lookup_identity', 'mod_intercept_form_submit', 'sssd-dbus'],
-    }
+    notify {'The --foreman-configure-ipa-repo option is deprecated and no longer needed. If you have used it before, disable and delete the repo.':}
```

Comment:
> I wanted it to be in reports coming from agents, we don't display master log anywhere AFAIK (in installer it does not matter that much)

That'd be unusual, and would continually result in an active resource.  Using the parser functions for deprecations is the form.

> the message should be changed a bit so it does not refer to installer argument but rather class parameter right?

Indeed.